### DISCUSSION TITLE
How to access current user details from custom interface and perform db queries?

### DISCUSSION THREAD

**Discussion by jakeceballos** [directus/directus#3774] [Archive]
I'm creating a custom interface to show select items pulled from an external API. How can I determine who the user is so that I can retrieve additional information from the database to perform the API request?

Thank you in advance!

**Comment by benhaynes** [comment]
@rijkvanzanten should have an answer for you, and then @philleepflorence can ensure this is added to the Docs for future reference.

**Comment by soyjimmysaenz** [comment]
@jakeceballos take a look:
https://github.com/soyjimmysaenz/directus-extensions-samples/tree/main/app-extensions/src/interfaces/user-info

My reference was `app/src/views/private/components/user-popover/user-popover.vue`

I hope it helps you.
